zsh/.zgen-setup

```diff
@@ -17,20 +17,20 @@ fi
 # is unclean.
 ZGEN_DIR=${ZGEN_DIR:-$HOME/.zgenom}
 
-if [[ ! -f $ZGENOM_PARENT_DIR/zgenom/zgenom.zsh ]]; then
+if [[ ! -f $ZGENOM_PARENT_DIR/.zgenom/zgenom.zsh ]]; then
```

Comment:
This will cause double cloning in existing installations unless you also check for the old path.

Reply:
Also, `~/.zgenom` is used by the zgenom framework to store the `git` clones of plugins used.

Reply:
Maybe clone it into `.zgenom-framework` or `.zqs-zgenom`? I think you're right, using a dot prefix will make it cleaner in listings / Finder windows.

Reply:
oh sorry. i'm new to zgenom, i was just trying to avoid multiple folders.

So, the idea is to have a completely separate configuration, even for those who already use zgenom?

And if so, it makes sense to create a `.zqs-zgenom`

Reply:
By default, zgenom clones all the plugins in your configuration into `~/.zgenom`, so we shouldn't clone zgenom itself there, it would make a very messy directory.

The quickstart kit is optimized for people who are going from no frameworks and plugins to a starter setup that should be as painless as possible to use. If someone already uses zgenom, they should know how to move their checkout to something compliant with the kit.

I do like the idea of making the zgenom checkout not visible by default in listings, I just want to keep things separated.

Reply:
:+1: i will update the PR.

Reply:
Sounds good, thanks. Have it check for the old name exists too and don't re-clone if that exists, but use something like `.zqs-zgenom` if it is cloning from scratch.